net-p2p/go-ipfs-bin/go-ipfs-bin-0.4.14.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit bash-completion-r1
+
+DESCRIPTION="Main implementation of IPFS"
+HOMEPAGE="https://ipfs.io/"
+SRC_URI="amd64? ( https://dist.ipfs.io/go-ipfs/v${PV}/go-ipfs_v${PV}_linux-amd64.tar.gz )
+	x86? ( https://dist.ipfs.io/go-ipfs/v${PV}/go-ipfs_v${PV}_linux-386.tar.gz )
+	arm? ( https://dist.ipfs.io/go-ipfs/v${PV}/go-ipfs_v${PV}_linux-arm.tar.gz )"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~arm ~x86"
+IUSE="+fuse"
+
+RDEPEND="fuse? ( sys-fs/fuse )"
+S="${WORKDIR}/go-ipfs"
+
+QA_PREBUILT="/usr/bin/ipfs"
+
+src_install() {
+	dobin ipfs
+
+	newbashcomp "${FILESDIR}/ipfs-completion.bash" "${PN}"
```

Comment:
```
 * Problems with installed bash completions were found:
 * 
 * 	go-ipfs-bin: incorrect name, no completions for 'go-ipfs-bin' command defined.
 * 	ipfs: missing alias (symlink) for completed command.
 * 
 * For more details on installing bash-completions, please see:
 * https://wiki.gentoo.org/wiki/Bash/Installing_completion_files

Reply:
About the file size, I mentioned it in PR #7092. It will be better to download it manually? In this case I think we will have problems with network-sandbox.

Reply:
Add it to SRC_URI.